File: 1074.py
n = int(input())
x =['']
for i in range(1,n + 1):
    x.append(int(input()))
    
for i in range(1,n + 1):
    if x[i] == 0:
        print('NULL')
        
    if x[i] > 0:
        if x[i] % 2 == 0:
            print('EVEN POSITIVE')
        else:
            print('ODD POSITIVE')
            
    if x[i] < 0:
        if x[i] % 2 == 0:
            print('EVEN NEGATIVE')
        else:
            print('ODD NEGATIVE')

File: CSharp/1074.cs
using System; 

class URI {

    static void Main(string[] args) 
    {
        int n = int.Parse(Console.ReadLine());
        int num;

        for(int i=0;i<n;i++)
        {
            num = int.Parse(Console.ReadLine());
            if(num>0 && num%2==0)
            {Console.WriteLine("EVEN POSITIVE");}
            else if(num>0 && num%2!=0)
            {Console.WriteLine("ODD POSITIVE");}
            else if(num<0 && num%2==0)
            {Console.WriteLine("EVEN NEGATIVE");}
            else if(num<0 &&num%2!=0)
            {Console.WriteLine("ODD NEGATIVE");}
            else{Console.WriteLine("NULL");}
            
        }
    }

}